addons/account/models/partner.py

```diff
@@ -603,6 +603,6 @@ def _merge_method(self, destination, source):
         """
         Prevent merging partners that are linked to already hashed journal items.
         """
-        if self.env['account.move.line'].search([('move_id.inalterable_hash', '!=', False), ('partner_id', 'in', source.ids)], limit=1):
+        if self.env['account.move.line'].sudo().search([('move_id.inalterable_hash', '!=', False), ('partner_id', 'in', source.ids)], limit=1):
```

Comment:
We need to also add `active_test=False`

Reply:
why ? the field active is not present on aml ?

Reply:
Indeed, I have no idea how I came up with this :thinking: